pkg/validation/validation.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package validation provides untils for lumberjack/data access logs
+// validation.
+package validation
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"google.golang.org/protobuf/encoding/protojson"
+
+	lepb "cloud.google.com/go/logging/apiv2/loggingpb"
+	cal "google.golang.org/genproto/googleapis/cloud/audit"
+)
+
+// ValidateLogFromFile validates a lumberjack log in the given file.
+func ValidateLogFromFile(path string) error {
```

Comment:
I think we can get rid of this. Would rather reading from stdin. See example https://github.com/abcxyz/jvs/blob/main/pkg/cli/token_validate.go#L47

Reply:
I tried that approach, it only reads the first line. probably due to https://github.com/abcxyz/pkg/blob/main/cli/command.go#L293

Reply:
Just don't use Prompt then. Prompt reads line. In this case we should read until EOF

Reply:
Will need to add a func in cli to read multi-line input.

Reply:
I will add one in lumberjack and probably move it in cli later.